plugins/woocommerce/client/blocks/.eslintrc.js

```diff
@@ -247,8 +247,24 @@ module.exports = {
 				'you-dont-need-lodash-underscore/omit': 'off',
 			},
 		},
+		{
+			files: [
+				'assets/js/**/test/**/*.{js,jsx,ts,tsx}',
+				'assets/js/**/*.test.{js,jsx,ts,tsx}',
+			],
+			parser: '@typescript-eslint/parser',
+			plugins: [ 'jest', '@typescript-eslint' ],
+			extends: [ 'plugin:jest/recommended' ],
+			rules: {
+				'jest/no-mocks-import': 'off',
```

Comment:
`plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/test/express-payment-methods.tsx` imports mocks. For now, let's keep this rule disabled.